docs/src/manual/models.md

```diff
@@ -0,0 +1,263 @@
+```@meta
+CurrentModule = JuMP
+DocTestSetup = quote
+    using JuMP, GLPK
+end
+DocTestFilters = [r"≤|<=", r"≥|>=", r" == | = ", r" ∈ | in ", r"MathOptInterface|MOI"]
+```
+
+# Models
+
+A JuMP model keeps a [MathOptInterface (MOI)](https://github.com/jump-dev/MathOptInterface.jl)
+*backend* of type `MOI.ModelLike` that stores the optimization
+problem and acts as the optimization solver. We call it an MOI *backend* and not
+optimizer as it can also be a wrapper around an optimization file format such as
```

Comment:
`it can also for instance be`. It can be more than just either an optimizer of a MOI.Formats model, here it seems you say these are the only 2 possibilities.

Reply:
Yeah I haven't touched this part. I'm going to move it to the end and simplify it a lot.